fendermint/eth/api/src/handlers/http.rs

```diff
@@ -20,7 +20,7 @@ pub async fn handle(
 
     // NOTE: Any authorization can come here.
 
-    tracing::debug!("RPC request: {request:?}");
+    tracing::info!("RPC request: {request:?}");
```

Comment:
Do we really want to log every request in full glory in everyday scenarios?

Reply:
In other RPC APIs from my past life it was useful to have all of the requests explicitly logged even if it was noisy, but that might just be my personal preference. I'll revert and once we deploy in production we can figure out the right balance.